Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer.js';
+import HeaderTitle from './components/HeaderComponents/HeaderTitle.js';
+import HeaderContent from './components/HeaderComponents/HeaderContent.js';
+import CardContainer from './components/CardComponents/CardContainer.js';
+import CardBanner from './components/CardComponents/CardBanner.js';
+import CardContent from './components/CardComponents/CardContent.js';
+import Footer from './components/FooterComponents/Footer.js';
+
```

Comment:
If you are just using the HeaderContainer and the CardContainer on this component, you only need to import thpse items.